pkg/server/app/models.go

```diff
@@ -237,3 +249,33 @@ func unsetEnvVars(app *App, evs []string) {
 		}
 	}
 }
+
+func newListResponse(list []*List) *appb.ListResponse {
+	if list == nil {
+		return nil
+	}
+
+	appNames := []*appb.ListResponse_App{}
+
+	for _, item := range list {
+		if item == nil {
+			continue
+		}
+		var tmp []string
+		for _, elt := range item.Addresses {
+			tmp = append(tmp, elt.Hostname)
```

Comment:
There's a different behavior with AWS and GCP, one returns the hostname and other returns ip, did you check it? related https://github.com/luizalabs/teresa-api/pull/162/files